README.md

```diff
@@ -162,18 +162,24 @@ class can notify you whenever any DJ Link devices on your network report
 the start of a new beat:
 
 ```java
-import org.deepsymmetry.beatlink.BeatFinder;
+
 
 // ...
 
-  BeatFinder.getInstance().addBeatListener(new BeatListener() {
-      @Override
-      public void newBeat(Beat beat) {
-         // Your code here...
-      }
-  });
+  BeatFinder.getInstance().
+
+addBeatListener(new BeatListener() {
+    @Override
+    public void newBeat (Beat beat){
+        // Your code here...
+    }
+});
+
+        BeatFinder.
+
+getInstance().
```

Comment:
Making a note for myself to fix this.

Reply:
Ooh, having some integration tests would definitely be a nice improvement. Can we somehow use a maven profile to control those so they don’t run by default, though, since people may not always be developing in a context where they have appropriate hardware available and connected?

Reply:
And oops, I don’t know how this got attached here, I was trying to thread it under your comment about integration tests, sorry.

Reply:
Yes, we can set a maven profile which will only run them by setting skipTests to true on the default and then creating another profile with skipTests as false. Ill set this up.